data/ssystem_minor.ini

```diff
@@ -448,6 +502,7 @@ absolute_magnitude             = 22.08
 albedo                         = 0.07
 coord_func                     = comet_orbit
 name                           = 1I/2017 U1 (`Oumuamua)
+iau_designation                = A/2017 U1
```

Comment:
This could be used as discovery_code. The official IAU designation is 1I/... Probably

name=`Oumuamua
iau_designation=1I/2017 U1
discovery_code=A/2017 U1

Reply:
`discovery_code` is used for comets only